src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import MenuItem from '@components/MenuItem';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import SpacerView from '@components/SpacerView';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWaitForNavigation from '@hooks/useWaitForNavigation';
+import Navigation from '@libs/Navigation/Navigation';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import FeatureEnabledAccessOrNotFoundWrapper from '@pages/workspace/FeatureEnabledAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function QuickbooksAdvancedPage({policy}: WithPolicyProps) {
+    const styles = useThemeStyles();
+    const waitForNavigate = useWaitForNavigation();
+    const {translate} = useLocalize();
+
+    const policyID = policy?.id ?? '';
+    const {autoSync, syncPeople, autoCreateVendor, reimbursementAccountID, collectionAccountID, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const {bankAccounts} = policy?.connections?.quickbooksOnline?.data ?? {};
+
+    const qboSyncToggleSettings = [
+        {
+            title: translate('workspace.qbo.advancedConfig.autoSync'),
+            subTitle: translate('workspace.qbo.advancedConfig.autoSyncDescription'),
+            isActive: Boolean(autoSync),
+            onToggle: () => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.AUTO_SYNC, !autoSync),
```

Comment:
```suggestion
            onToggle: () => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.AUTO_SYNC, {enabled: !autoSync?.enabled}),
```

The `autoSync` field is an object

Reply:
@s77rt it doesn't seem like we have it all set, autoSync is boolean in the screenshot, although it reflects as an object in types. Just had to
<img width="200" alt="Screenshot 2024-04-12 at 14 24 58" src="https://github.com/Expensify/App/assets/22779249/69c79682-d06a-43cb-bdc0-a3969820a100">
 leave it there for now and was avoiding too many changes with the types cc @hayata-suenaga

Reply:
@teneeto, where did you get the `connections` value in the screenshot? Is that the screenshot I sent before?

Reply:
No, This was a log on `policy`

Reply:
This one is still unresolved

Reply:
@teneeto I think we should rely on types as source of truth. The onyx data is showing autoSync as boolean because the above code overwrote it

Reply:
Done